pkg/network/nmstate.go

```diff
@@ -14,14 +15,18 @@ import (
 
 func changeSafeNMState(prev, next *opv1alpha1.NetworkAddonsConfigSpec) []error {
 	if prev.NMState != nil && !reflect.DeepEqual(prev.NMState, next.NMState) {
-		return []error{errors.Errorf("cannot modify NMState state handler configuration once it is deployed")}
+		if !(prev.NMState != nil && next.NMState == nil) {
+			return []error{errors.Errorf("cannot modify NMState state handler configuration once it is deployed")}
```

Comment:
Now when we support removal of knmstate, we don't need this check anymore. You can drop the whole function.

Reply:
hmm but arent there updates which arent legit ? (where deepequal might show a real error)
i understand, just making sure please

Reply:
bump

Reply:
think we removed it, so its old